mantidproject/mslice
# Implement plot options dialog

Implemented Plot Options Dialog. The dialog allows users to change modify the plots
title, x axis label, y axis label, legends, x axis range, y axis range and to toggle errorbars.
For convenience a single button to toggle error bars is on the toolbar

**To test:**
1. Bring up a cut on mslice.
2. Try toggling the legends with the toggle legends button.
3. Open the plot options dialog.
4. Try hiding/showing the errorbars by toggling the checkbox and clicking ok.
5. Try hiding/showing the entire legend box using the checkbox labelled `Show legends`
6. Bring up a cut with multiple entries (by overplotting or using the `width` parameter)
7. Try hiding/showing single legend entries with the checkboxes in each legend entry.
8. Try changing the figure title, x axis label and y axis label.
9. Bring up a slice. Verify the plot options dialog does not contain `Show Errorbars` checkbox and does not contain the `Legends` groupbox

Fixes #61
